src/main/java/mekanism/common/integration/OreDictManager.java

```diff
@@ -66,9 +66,14 @@ public static void init() {
 			RecipeHandler.addEnrichmentChamberRecipe(StackUtils.size(ore, 1), new ItemStack(MekanismItems.OtherDust, 4, 1));
 		}
 
-		if (OreDictionary.getOres("itemRubber").size() > 0) {
+		if (OreDictionary.getOres("itemRawRubber").size() > 0) {
```

Comment:
Why not use `!OreDictionary.getOres("itemRubber").isEmpty()` instead?

Reply:
well i'm lazy and didnt bothered to change that?, does the same anyways.

Reply:
feel free to change it

Reply:
Alright, I'll eventually clean up the code in future commits